Jenkinsfile

```diff
@@ -105,16 +105,8 @@ pipeline {
 }
 
 /**
- * @param version version to test
- * @return true, if given version does not contain SNAPSHOT.
- */
-def isRelease(version) {
-    echo version
-    return !version.contains('SNAPSHOT')
-}
-
-/**
- * @return version specified in gradle.properties
+ * @return Version specified in gradle.properties. Fails if master contains a release version (to prevent pushing
```

Comment:
nit: If this is javadoc, then Verbose main fragment is preferred before naked `@return` statement.

Reply:
Thanks. Done.